src/common/providers/logger/logger.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { LoggerService } from './logger.service';
+const nestLoggerMock = {
+  error: jest.fn(),
+  log: jest.fn(),
+  warn: jest.fn(),
+  debug: jest.fn(),
+  verbose: jest.fn(),
+};
+
+describe('LoggerService', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  ['error', 'log', 'warn', 'debug', 'verbose'].forEach(level => {
+    it(`should call the ${level} method of the Nest Logger`, () => {
+      const logger = new LoggerService(nestLoggerMock as any, {
+        get: () => 'true',
+      });
+      logger[level]('message', {});
+      expect(nestLoggerMock[level]).toHaveBeenCalledWith('message');
+    });
+  });
```

Comment:
You can use the jest `each` syntax.
As an example:
```
test.each([
    [true, true, true, 'activities-city-sport'],
    [true, true, false, 'activities-city-sport'],
    [true, false, true, 'activities-region-sport'],
    [true, false, false, 'activities-sport'],
    [false, true, true, 'activities-city'],
    [false, true, false, 'activities-city'],
    [false, false, true, 'activities-region'],
    [false, false, false, 'activities'],
  ])('hasSport %s + isPlaceCity %s + isPlaceRegion %s = %s', (hasSport, isPlaceCity, isPlaceRegion, routeName) => {
    expect(getRouteNameToPush({
      hasSport: hasSport,
      isPlaceCity: isPlaceCity,
      isPlaceRegion: isPlaceRegion,
    })).toBe(routeName)
  })```
I think you can replace the `test` by `it`, it should be an alias.